crates/wallet/src/wallet/mod.rs

```diff
@@ -1216,7 +1216,7 @@ impl Wallet {
 
     /// Return the balance, separated into available, trusted-pending, untrusted-pending and immature
```

Comment:
```suggestion
    /// Returns the balance, separated into available, trusted-pending, untrusted-pending and immature
```

Since we are fixing this anyways let's adjust the docstring?